src/core/jsproxy.c

```diff
@@ -1533,8 +1578,30 @@ JsProxy_init(PyObject* core_module)
 {
   bool success = false;
 
+  PyObject* _pyodide_core = NULL;
+  PyObject* jsproxy_mock = NULL;
   PyObject* asyncio_module = NULL;
 
+  _pyodide_core = PyImport_ImportModule("_pyodide._core");
+  FAIL_IF_NULL(_pyodide_core);
+  _Py_IDENTIFIER(JsProxy);
+  jsproxy_mock =
+    _PyObject_CallMethodIdObjArgs(_pyodide_core, &PyId_JsProxy, NULL);
+  FAIL_IF_NULL(jsproxy_mock);
+
+#define SET_DOCSTRING(x)                                                       \
+  FAIL_IF_MINUS_ONE(set_method_docstring(&x, jsproxy_mock))
+  SET_DOCSTRING(JsProxy_object_entries_MethodDef);
+  SET_DOCSTRING(JsProxy_object_keys_MethodDef);
+  SET_DOCSTRING(JsProxy_object_values_MethodDef);
```

Comment:
Could you please add a comment about the effect of these `SET_DOCSTRING` macros?